recipe/bld.bat

```diff
@@ -2,16 +2,15 @@ rmdir /s /q build
 mkdir build
 cd build
 
-:: Only GENERIC Target is supported on MSVC
-:: See https://github.com/giaf/blasfeo/blob/b6bc34c6cb1995aea60968403016110bd3288e3b/CMakeLists.txt#L255
 cmake ^
     -G "Ninja" ^
+    -DCMAKE_C_COMPILER=clang-cl ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
     -DCMAKE_BUILD_TYPE=Release ^
     -DBUILD_TESTING:BOOL=ON ^
-    -DBLASFEO_TESTING:BOOL=ON ^
+    -DBLASFEO_TESTING:BOOL=OFF ^
```

Comment:
Given that we disable this, did you tested that an executable on Windows can consume the produced libraries well?

Reply:
I disabled this because of the potential arm build, but I can add it back and see if the CI has trouble there.

Reply:
If the problem is arm, we can add an if when we will add win-arm64 support.